### PR TITLE
[10.0] [MIG] l10n be partner identification lmi

### PR DESCRIPTION
**Pull request by lmignon**

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code Review

File: l10n_be_partner_identification/tests/test_l10n_be_partner_identification.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#  ©  2016 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import openerp.tests.common as common
```
Comment (rousseldenis) [inline l10n_be_partner_identification/tests/test_l10n_be_partner_identification.py:4]: Oups! Didn't pay attention to this...

**Review by rousseldenis — CHANGES_REQUESTED** [review]

**Comment by sbidoul** [comment]
@lmignon @rousseldenis regarding this PR and #42, should we have one module per identification number (in which case, the module name in this PR must suffixed with `rrn` or something). Or create one module for all be identification schemes? In which case #42 must be merged into this one.

**Comment by rousseldenis** [comment]
IMO, it's quite difficult to identify every belgian scheme (unknown ones). So, by case, we create specialized module based on partner-contact repo one. Suffixing this one with rrn should be the solution.

**Pull request closed without merging**